docs/program/SDK-as-client.md

```diff
@@ -186,6 +192,23 @@ main();
 {{% /tab %}}
 {{< /tabs >}}
 
+{{% alert title="Caution" color="caution" %}}
```

Comment:
This should stay above because it's important people see it - moving it here hides it a bit

Reply:
cool cool, lgtm!